### PR TITLE
chore: run express generator

### PR DESCRIPTION
**Pull request by DonLe27**

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "server",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www"
+  },
+  "dependencies": {
+    "cookie-parser": "~1.4.4",
```
Comment (BryanPan342) [inline server/package.json:9]: do we need node as a dependency?

**Review by BryanPan342 — COMMENTED** [review]
I dont know enough about it but can we have a README that walks through the file types and 

I see that we start at `bin/www` and from there the express code gets loaded from `app.js` but how does the `.jade` files come into play?

**Review by josephtclegg — APPROVED** [review]

**Comment by BryanPan342** [comment]
this is now stale?

**Comment by arjunr00** [comment]
^closing.

**Pull request closed without merging**